config_defaults_inc.php

```diff
@@ -1784,11 +1784,15 @@
 
 /**
  * html tags
- * Set this flag to automatically convert www URLs and
- * email addresses into clickable links
+ * This flag controls whether www URLs and email addresses
+ * are automatically converted to clickable links as well as
+ * where the www links open when clicked. Options:
+ * 0 = Do not convert www URLs or email addresses to links
+ * 1 = Convert to links that open in the current window (DEFAULT)
+ * 2 = Convert to links that open in a new window
  * @global integer $g_html_make_links
  */
-$g_html_make_links = ON;
+$g_html_make_links = 1;
```

Comment:
Don't use "magic numbers". Please define constants as appropriate (in core/constants_inc.php).